doc/credits.rst

```diff
@@ -0,0 +1,40 @@
+*******************
+Authors and Credits
+*******************
+
+Core Package Contributors
+=========================
+
+* Han Wang
+* Jinzhe Zeng
+* Marián Rynik
+* denghuilu
+* Denghui Lu
+* Lu
+* GeiduanLiu
+* tuoping
+* ziyao
+* Duo
+* YWolfeee
+* marian-code
+* ZiyaoLi
+* haidi
+* Jiequn Han
+* Yixiao Chen
+* bwang-ecnu
+* hlyang
+* Linfeng Zhang
+* deepmodeling
+* iProzd
+* njzjz
+* wsyxbcl
```

Comment:
It is messy to use a mixed list of real names and github IDs. Could you please clean it up?
Some of the item are duplicated, Marian Rynic and marian-code is the same person. Duo and iprozd are the same person.

Reply:
Sorry. It's fixed now.

Reply:
I suggest showing the contributors in an alphabetical order

Reply:
Of course, Sorry.